slc/tools/leon2mt-fpga-ctl.in

```diff
@@ -130,7 +130,9 @@ while True:
 
     output = splitBuffer(mybuf)
     sys.stdout.write(output[0])
+    sys.stdout.flush()
```

Comment:
To make sure output is displayed immediately; by default I'm only seeing flushes on newlines, which is pretty confusing while debugging.